samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/ItemBean.java

```diff
@@ -1274,6 +1275,22 @@ public void setSerializedImageMap(String serializedString) {
 		}
 	}
 
+  /**
+   * ￼ * for fib questions
+   *
+   * @param param the pair of markers
+   */
+  public void setMarkersPair(String param) {
+    String paramTmp = param;
+    if ((paramTmp == null) || (paramTmp.length() < 2)) paramTmp = "{}";
```

Comment:
We are not allowed to, for example, set just one char, so if we try it, markers are stabilished as default {}. Should we remove de length < 2 check, so we can show the error properly error message?

Reply:
i think this is okay